### PR TITLE
7.0 ci

### PR DESCRIPTION
**Pull request by StefanRijnhart**
Like https://github.com/OCA/banking/pull/1, but against the correct branch

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+virtualenv:
+  system_site_packages: true
+
+install:
+ - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
+ - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+ - travis_install_nightly 7.0
+ - pip install coveralls flake8
+ - pip install unidecode
```
Comment (bwrsandman) [inline .travis.yml]: BeautifulSoup

**Comment by StefanRijnhart** [comment]
BeautifulSoup is used in a deprecated module. Can we exclude such modules from travis?

**Comment by bwrsandman** [comment]
Move them to __unported__/

**Comment by StefanRijnhart** [comment]
Well, I am not Odoo and I like to leave my deprecated modules in for the stable release during which they were deprecated... I added the dependency.

**Pull request merged**